### PR TITLE
 API endpoints to send file to multiple nodes in the network.

### PR DESCRIPTION
**Pull request by tewarig**
This solves #20 partially
currently, 
- [x]  we need to hard code the address of nodes in the network
- [x]  it sends the file as a whole to the other nodes in the network.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -137,6 +137,15 @@ def upload():
             file.save(destination)
             os.system('python utils/encryption.py')
             flash("File Uploaded", "success")
+             //this is a temp hardcoded value for node we will change it later. 
+            test_response = requests.post( 'http://localhost:3000/upload' , files = {"form_field_name": file})
```
Comment (pkesavap) [inline app.py:141]: Correct me if im wrong,
the place where localhost link is given, if we parse other nodes ip's , would the data be post across them all?

File: app.py [hunk 2]
```diff
@@ -99,6 +99,28 @@ def logout():
     flash("You are Logged out","danger")
     return redirect(url_for('login'))
 
+@app.route("/imageUpdate",methods=["POST"])
+def imageUpdate():
```
Comment (pkesavap) [inline app.py:103]: this method is similar to def Upload() below, any reason you reproduced it again?

Also this method expects a template to be returned in the last line.

**Review by pkesavap — CHANGES_REQUESTED** [review]
Good start!
Please remove all the binary files out of the PR.

I intend to remove them from the upstream as well.

File: app.py (hunk 1 above)
Comment (tewarig) [reply app.py:141]: yes i would work .

File: app.py (hunk 2 above)
Comment (tewarig) [reply app.py:103]: actually, this method provides a route to files that can be uploaded from other nodes. i would be removing last line where is expects to return a template.

**Comment by pkesavap** [comment]
Please also squash all the commits into a single one for each implemntation.
It will be easier to review.

File: app.py [hunk 3]
```diff
@@ -6,25 +6,18 @@
 from functools import wraps
 import requests
 import os
-
```
Comment (pkesavap) [inline app.py:9]: You can leave a space here

File: app.py [hunk 4]
```diff
@@ -6,25 +6,18 @@
...
 app = Flask(__name__)
-
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///db.user_sqlite3'
-
 db = SQLAlchemy(app)
-
```
Comment (pkesavap) [inline app.py:16]: please leave a space here too

File: app.py [hunk 5]
```diff
@@ -6,25 +6,18 @@
...
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(50))
     username = db.Column(db.String(50), nullable=False)
     email = db.Column(db.String(50), nullable=False)
     password = db.Column(db.String(300), nullable=False)
-
-
 app.config['SECRET_KEY'] = '1234567DHTCONSOLE'
 APP_ROOT = os.path.dirname(os.path.abspath(__file__))
-
```
Comment (pkesavap) [inline app.py:27]: You can leave a space here as well

**Review by pkesavap — CHANGES_REQUESTED** [review]
You can also remove binary files, images venv files from this PR

File: app.py (hunk 4 above)
Comment (tewarig) [reply app.py:16]: ok

File: app.py (hunk 5 above)
Comment (tewarig) [reply app.py:27]: ok

File: app.py (hunk 3 above)
Comment (tewarig) [reply app.py:9]: ok

**Comment by pkesavap** [comment]
Thanks @tewarig much better!

**Pull request merged**